tests/reports.js

```diff
@@ -3,11 +3,11 @@ var assert = require("assert")
 var date = require("./util/date");
 
 step("Select start date, end date and <reportFormat> format for <reportName> and click on run button", async function (reportFormat, reportName) {
-	let startDate = date.today()
+	let startDate = date.yesterday()
 	let endDate = date.today()
 	await timeField(toRightOf(text(reportName)), below(text("Start Date"))).select(startDate)
 	await timeField(toRightOf(text(reportName)), toRightOf(text("Start Date")), below(text("End Date"))).select(endDate)
-	await dropDown(toRightOf(text(reportName)), below(text("Format"))).select(reportFormat)
+	await dropDown(toRightOf(text("End Date")), below(text("Format"))).select(reportFormat)
```

Comment:
We need to select report format for the repot which we are going to validate. This is not the right way to do it.